apps/web/src/views/Swap/MMLinkPools/hooks/useSwapCallArguments.ts

```diff
@@ -21,9 +21,9 @@ export interface SwapCall {
  * @param recipientAddressOrName
  */
 export function useSwapCallArguments(
-  trade: SmartRouterTrade<TradeType> | null, // trade to execute, required
+  trade: SmartRouterTrade<TradeType>, // trade to execute, required
```

Comment:
the value is probably null?

Reply:
I think the type assertion thing I mentioned before means to add `undefined` to the type definition so no need for setting the type explicitly

`trade: SmartRouterTrade<TradeType> | null | undefined`